tests/e2e/Adapter_Installer/composer.json

```diff
@@ -1,6 +1,6 @@
 {
     "require-dev": {
-        "phpunit/phpunit": "^8"
+        "phpunit/phpunit": "^9.3"
```

Comment:
All fixtures have to use the new version too, because these versions have incompatible interfaces, and they cause failures for E2ETest.